### ISSUE TITLE
Change loss functions / activate batch normalisation 

### ISSUE THREAD

**Issue by vanessacerrone** [javivilladamigo/ImageRecolorization#6]

**Comment by vanessacerrone** [comment]
Compare results
